test/test_primitives.jl

```diff
@@ -85,6 +87,12 @@ using Test
     GeoInterface.getfeature(::FeatureCollectionTrait, fc::MyFeatureCollection) = fc.geoms
     GeoInterface.getfeature(::FeatureCollectionTrait, fc::MyFeatureCollection, i::Integer) = fc.geoms[i]
 
+    GeoInterface.israster(::Type{<:Raster}) = true
+    GeoInterface.trait(::Raster) = RasterTrait()
+    GeoInterface.extent(::RasterTrait, ::Raster) = Extents.Extent()
+    GeoInterface.crs(::RasterTrait, ::Raster) = ""
```

Comment:
Should `crs` be a GeoFormatTypes.jl wrapper or `nothing` ? 

How do we interpret strings

Reply:
In the vector spec we say it should be a GFT thing.

Reply:
But this object is passing the test suite